Formula/zbctl.rb

```diff
@@ -7,6 +7,11 @@ class Zbctl < Formula
   license "Apache-2.0"
   head "https://github.com/camunda/zeebe.git", branch: "develop"
 
+  livecheck do
```

Comment:
```suggestion
  # Upstream creates stable version tags (e.g., `v1.2.3`) before a release but
  # the version isn't considered to be released until a corresponding release
  # is created on GitHub, so it's necessary to use the `GithubLatest` strategy.
  livecheck do
```

I make sure to leave an explanatory comment before the `livecheck` block when we're using the `GithubLatest` strategy but the `stable` URL isn't a release asset (e.g., `.../releases/download/v1.2.3/example-1.2.3.tar.xz`). Otherwise, we run a risk of someone removing the `livecheck` block thinking that it's not necessary based on `stable` (and/or we have to spend time researching the situation each time we come across this `livecheck` block to confirm it's needed).